src/renderer/component/publishForm/internal/name-help-text.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from 'react';
+import Button from 'component/button';
+import { buildURI } from 'lbry-redux';
+import type { Claim } from 'types/claim';
+
+type Props = {
+  uri: ?string,
+  myClaimForUri: ?Claim,
+  isStillEditing: boolean,
+  onEditMyClaim: (any, string) => void,
+};
+
+class NameHelpText extends React.PureComponent<Props> {
+  render() {
+    const { uri, myClaimForUri, onEditMyClaim, isStillEditing } = this.props;
+
+    let nameHelpText;
+
+    if (isStillEditing) {
+      nameHelpText = __(
+        'You are currently editing this claim. If you change the URL, you will need to reselect a file.'
+      );
+    } else if (uri && myClaimForUri) {
+      const editUri = buildURI({
+        contentName: myClaimForUri.name,
+        claimId: myClaimForUri.claim_id,
+      });
+
+      nameHelpText = (
+        <React.Fragment>
```

Comment:
Can also use the short syntax (empty tags):
```
<>
  {/* markup here */}
</>
```

Reply:
Do we need to do something with webpack for that to work? Doesn't seem to build properly when I do that.